vite.config.js

```diff
@@ -1,11 +1,13 @@
-import { fileURLToPath, URL } from 'url';
 
 import { defineConfig } from 'vite';
 import Vue from '@vitejs/plugin-vue';
 import Vuetify from '@vuetify/vite-plugin';
+import cesium from 'vite-plugin-cesium';
```

Comment:
This plugin is very cool, I'm glad we don't have to deal with all that webpack configuration this time around :)